cli/src/cli_util.rs

```diff
@@ -2547,10 +2547,19 @@ pub struct EarlyArgs {
     #[arg(long, value_name = "WHEN", global = true)]
     pub color: Option<ColorChoice>,
     /// Disable the pager
-    #[arg(long, value_name = "WHEN", global = true, action = ArgAction::SetTrue)]
+    #[arg(long, global = true, action = ArgAction::SetTrue)]
     // Parsing with ignore_errors will crash if this is bool, so use
     // Option<bool>.
     pub no_pager: Option<bool>,
+    /// Override the configured key to be used for the commit signing.
+    /// Is only used when a commit signing operation has to be performed by the
+    /// operation.
+    #[arg(long, value_name = "KEY", global = true)]
+    pub sign_with: Option<String>,
+    /// Don't sign unsigned commits when configured to sign all, is ignored
+    /// otherwise.
+    #[arg(long, global = true, action = ArgAction::SetTrue)]
+    pub no_sign: Option<bool>,
```

Comment:
(just a comment) These options don't have to be parsed "early", but they are copied to the `config_toml` array. So it's probably okay to belong to `EarlyArgs`.